### ISSUE TITLE
GIDS Upload Page - Form elements are missing labels

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#4550]
When attempting to upload a new version of any file in GIDS, the user is presented with the file upload page. Three of the fields on this page are missing labels and are therefore not being properly read by a screen reader. 
- Csv file type
- Skip lines before header
- Column separator

Labels must be added to these fields to ensure section 508 compliance.

![Screen Shot 2020-01-03 at 3.08.40 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/3075dea7-ca3f-4ae3-8449-715e6a92864c)

This defect can be reproduced using any of the following tools:
- [Axe](https://www.deque.com/axe/)
- [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html)
- [Wave](https://wave.webaim.org/)

**Comment by dmccurdybah** [comment]
Can be tested on gibct-data-service branch `bah-label-fix`

PR: https://github.com/department-of-veterans-affairs/gibct-data-service/pull/541

**Comment by sonntag-adam** [comment]
Form elements are missing labels fixed
![Screen Shot 2020-01-10 at 10.04.16 AM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/7ceeeacf-3c6c-40f4-ad21-ba5a98a0face)

**Comment by sonntag-adam** [comment]
In staging, closing bug.
